app/views/jobseekers/profiles/preview/show.html.slim

```diff
@@ -0,0 +1,62 @@
+- content_for :page_title_prefix, t(".page_title")
+
+- content_for :breadcrumbs do
+  = govuk_back_link text: t("buttons.back"), href: jobseekers_profile_path
+
+.govuk-grid-row
+  .govuk-grid-column-two-thirds
+    h1.govuk-heading-xl class="govuk-!-margin-bottom-3" = t(".page_title")
+    p class="govuk-!-margin-bottom-3" = "QTS awarded in #{@profile.qualified_teacher_status_year}."
+
+    h2.govuk-heading-m class="govuk-!-padding-bottom-3"
+      = t(".job_preferences")
+    dl.govuk-summary-list
```

Comment:
We can use the summary list component here
https://govuk-components.netlify.app/components/summary-list/